app/src/app/[lng]/settings/page.tsx

```diff
@@ -48,8 +48,10 @@ export default function Settings({
   const { data: userInfo, isLoading: isUserInfoLoading } =
     api.useGetUserInfoQuery();
 
+  let isVoid: void;
+
   const { data: cities, isLoading: isCitiesLoading } = api.useGetCitiesQuery(
-    null,
+    isVoid,
```

Comment:
You can just remove this line and the definition above, void in the redux route type make it so you don't have to pass anything here.